src/ProgressMeter.jl

```diff
@@ -203,6 +237,34 @@ function updateProgress!(p::ProgressThresh; showvalues = Any[], valuecolor = :bl
     end
 end
 
+function updateProgress!(p::ProgressOnline; showvalues = Any[], valuecolor = :blue)
+    t = time()
+    if p.done
+        if p.printed
+            dur = durationstring(t-p.tfirst)
+            msg = @sprintf "%s %d \t Time: %s" p.desc p.counter dur
+            move_cursor_up_while_clearing_lines(p.output, p.numprintedvalues)
+            printover(p.output, msg, p.color)
+            printvalues!(p, showvalues; color = valuecolor)
+            println(p.output)
+        end
+        return
+    end
+
+    if t > p.tlast+p.dt
+        dur = durationstring(t-p.tfirst)
+        msg = @sprintf "%s %d \t Time: %s" p.desc p.counter dur
+        move_cursor_up_while_clearing_lines(p.output, p.numprintedvalues)
+        printover(p.output, msg, p.color)
+        printvalues!(p, showvalues; color = valuecolor)
+        # Compensate for any overhead of printing. This can be
+        # especially important if you're running over a slow network
+        # connection.
+        p.tlast = t + 2*(time()-t)
+        p.printed = true
+    end
```

Comment:
Add `return`? Just for return-type stability.

Reply:
Good catch. Added it.